### PR TITLE
Refactor the file structure

### PR DESCRIPTION
**Pull request by hbgxsm95**
Refactor the whole file structure under /iot. A directory called `sensor` is for the first sub-project. The other directory called `protocol` is for the second sub-project.
The files for the protocol are all under the directory `protocol`. The main program is under the ./protocol/examples.

### CODE REVIEW

File: go/iot/protocol/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+### Introduction
+
+When designing the IoT system structure, developers would always prefer the master/slave topology with the consideration
+of the costs. In this topology, there would be multiple slave nodes that consist of cheaper MCUs and connect to the
+master node, which would always be more powerful and expensive. In this way, we could improve the hardware resource
+utilization and decrease costs since slave nodes supporting simple hardware interface would be adequate for the sub-tasks.
+
+In this project, I design a protocol for the IoT systems with the above topology. It allows the slave nodes to transmit
```
Comment (songy23) [inline go/iot/protocol/README.md]: nit: s/design/designed

File: go/iot/protocol/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+the collected information to the master node which runs the openCensus framework. Systems that supports this protocol
```
Comment (songy23) [inline go/iot/protocol/README.md]: nit: s/openCensus/OpenCensus

File: go/iot/sensor/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+### Introduction
+
+This sub-project, as a prototype application of openCensus on the IoT industry, builds a demo for applying the
```
Comment (songy23) [inline go/iot/sensor/README.md]: nit: s/openCensus/OpenCensus

File: go/iot/sensor/README.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+openCensus framework on the IoT platform. In this project, we use the Raspberry Pi to collect data from some sensors and
```
Comment (songy23) [inline go/iot/sensor/README.md]: ditto

File: go/iot/sensor/README.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+export them to the back-end stackDriver server for visualization and persistence.
```
Comment (songy23) [inline go/iot/sensor/README.md]: s/stackDriver/StackDriver

File: go/iot/protocol/configure.sh [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+go get -u go get -u go.opencensus.io
+
+sudo apt-get install sshpass
+
+cd $(go env GOPATH)/src/github.com/census-ecosystem/oepncensus-experiments/go/iot/protocol
+
+sudo apt-get install gcc-arm-linux-gnueabihf
+
+chmod u+x ./run.sh
+
```
Comment (songy23) [inline go/iot/protocol/configure.sh]: Do you need to init and configure Cloud credentials in this script?

Comment (hbgxsm95) [reply go/iot/protocol/configure.sh]: solved.

**Pull request merged**